bin/codacy-analysis-cli.sh

```diff
@@ -17,14 +17,6 @@ log_error() {
   exit 3
 }
 
-test_docker_socket() {
```

Comment:
This function is called [here](https://github.com/codacy/codacy-analysis-cli/pull/458/files#diff-dc7fa8c766c6d2c527ab7ed606fd8d35048ede2185c4d2f1a5baa8b8c51480f3R112). If you delete this it will crash.
Did you test the new script?
What you want is to tweak the function to check that the suggested env variable (maybe `SKIP_DOCKER_SOCKET_CHECK`) contains `true`.